### PR TITLE
Update Min nodejs in localfs install/upgrade docs

### PR DESCRIPTION
**Pull request by hardillb**
## Description

Update to require NodeJS 18 to keep up with dependencies

## Related Issue(s)

https://github.com/FlowFuse/flowfuse/pull/3300

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [x] Documentation has been updated
    - [x] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Pezmc** [comment]
18 will have LTS until April 2025, so 👍 from me

**Review by Pezmc — APPROVED** [review]

File: docs/install/local/README.md
Comment (Steve-Mcl) [inline docs/install/local/README.md:260]: Is this referring to FlowFuse or localfs Ben?

Comment (hardillb) [reply docs/install/local/README.md:260]: I don't understand the question Steve. This is on the FlowFuse localfs install page.

Comment (Steve-Mcl) [reply docs/install/local/README.md:260]: Aaaarrrrrgggghhhh, complete brain f*** - forgot localfs was part of FlowFuse 🤦

**Comment by Steve-Mcl** [comment]
@hardillb Am I right in assuming that this PR is to coincide with imminent FF v2 release?

Currently, FF engines entry is fixed at 16.x and I dont see a PR for updating the [engines entry](https://github.com/FlowFuse/flowfuse/blob/d5d4ed6548cedccf851995d4c50c4a745114a900/package.json#L162)

Should that update be included here (or have I simply missed a PR or open issue that is taking care of that)?

**Comment by hardillb** [comment]
@Steve-Mcl  I've bumped the engine entry to match, we also need to check what version the tests are running on as I think they are 16 based at the moment.

**Comment by Steve-Mcl** [comment]
> we also need to check what version the tests are running on as I think they are 16 based at the moment.

It does appear to be 16.x in the test runner output Ben.

shall i hang fire on this or merge on test pass?

**Comment by hardillb** [comment]
@Steve-Mcl I've bumped the version used in the tests, so lets see what happens

**Review by Steve-Mcl — APPROVED** [review]
👍

**Comment by Steve-Mcl** [comment]
@hardillb I have approved but am unsure if there is a desired merge window - will leave that to you.

**Comment by knolleary** [comment]
I think this is good to merge now?

**Comment by hardillb** [comment]
@knolleary yep

**Pull request merged**
